pkg/base/k8s/kubeclient.go

```diff
@@ -104,7 +104,7 @@ func (k *KubeClient) CreateCR(ctx context.Context, name string, obj runtime.Obje
 		"requestUUID": requestUUID.(string),
 	})
 	crKind := obj.GetObjectKind().GroupVersionKind().Kind
-	ll.Infof("Creating CR %s: %v", crKind, obj)
```

Comment:
Is it possible to not print raw data at all, but `spec` field with INFO level?

Reply:
The requirement was to print debug with raw. The requirement was changed?

Reply:
+ labels and annotations, if possible

Reply:
Can you please write the requirement clearly - what should be done here? Because currently there are 3 options about what is needed to do with this log

Reply:
Pushed changed with logging CSI Objects fields - labels, annotations and spec at info level